rhesus-monkeys/scripts/convert-samples.py

```diff
@@ -0,0 +1,119 @@
+import polars as pl
+import re
+from pathlib import Path
+import polars.selectors as cs
+
+# Set the folder path for raw-data
+resource_dir = Path(__file__).resolve().parent.parent
+folder_path = resource_dir / "data-raw"
+unzip_path = folder_path / "unzipped"
+
+# Transpose all files with metabolite data
+def transpose_data(load_file_name, save_file_name, monkey_type):
+    df_1 = pl.read_csv(unzip_path / load_file_name)
+    df_1a = df_1.transpose(include_header=True, header_name=monkey_type, column_names="Metabolite")
+    df_1a.write_csv(folder_path / save_file_name)
+
+load_file_names = [
+    "concentration_infant_blood.csv",
+    "concentration_infant_urine.csv",
+    "concentration_maternal_blood.csv",
+    "concentration_maternal_placenta.csv",
+    "concentration_maternal_urine.csv"
+]
+save_file_names = [
+    "data_concentration_infant_blood.csv",
+    "data_concentration_infant_urine.csv",
+    "data_concentration_maternal_blood.csv",
+    "data_concentration_maternal_placenta.csv",
+    "data_concentration_maternal_urine.csv"
+]
+monkey_types = [
+"infant_sample_id",
+"infant_sample_id",
+"adult_sample_id",
+"adult_sample_id",
+"adult_sample_id"
+]
+for load_file_name, save_file_name, monkey_type in zip(load_file_names, save_file_names, monkey_types):
+    transpose_data(load_file_name, save_file_name, monkey_type)
```

Comment:
Rather than writing out the file names manually, use code to get all the files with `Path.glob()` [here](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob), rename the files in that list of files to snake case, and generate the monkey type based on the file name.